partiql-logical-planner/src/lower.rs

```diff
@@ -1922,7 +1922,7 @@ fn lit_to_lit(lit: &Lit) -> Result<logical::Lit, AstTransformError> {
         Lit::FloatLit(f) => logical::Lit::Double(OrderedFloat::from(*f as f64)),
         Lit::DoubleLit(f) => logical::Lit::Double(OrderedFloat::from(*f)),
         Lit::BoolLit(b) => logical::Lit::Bool(*b),
-        Lit::EmbeddedDocLit(s) => {
+        Lit::EmbeddedDocLit(s, _) => {
```

Comment:
Should the type argument be used in the following line rather than the hard-coded `Ion`?

Reply:
Eventually, but there's not enough in place to do so meaningfully in the PR.